contrib/storage-hive/core/src/test/java/org/apache/drill/exec/hive/HiveTestBase.java

```diff
@@ -17,28 +17,53 @@
  */
 package org.apache.drill.exec.hive;
 
+import java.io.File;
+import java.util.UUID;
+
+import org.apache.commons.io.FileUtils;
 import org.apache.drill.PlanTestBase;
 import org.apache.drill.exec.store.hive.HiveTestDataGenerator;
+import org.apache.drill.test.BaseDirTestWatcher;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
+import org.junit.runner.Description;
 
 /**
  * Base class for Hive test. Takes care of generating and adding Hive test plugin before tests and deleting the
  * plugin after tests.
  */
 public class HiveTestBase extends PlanTestBase {
-  protected static HiveTestDataGenerator hiveTest;
+
+  public static final HiveTestFixture HIVE_TEST_FIXTURE;
+
+  static {
+    // generate hive data common for all test classes using own dirWatcher
+    BaseDirTestWatcher generalDirWatcher = new BaseDirTestWatcher() {
+      {
+        String uniqueSuffix = String.format("_%s", UUID.randomUUID().toString().replace("-", ""));
```

Comment:
Why these replacements are needed?

Reply:
This suffix will be added to name of main directory created  in DirWatcher's starting() method, and in my opinion it's better to have directory name without dashes.

Reply:
I think dashes are ok, we use the same approach with CTTAS.